lib/device_api/android/device.rb

```diff
@@ -54,14 +59,14 @@ def status
       end
       
       def connect
-        ADB.connect(@ip_address, @port)
+        ADB.connect(ip_address, port)
```

Comment:
Out of interest, is there a reason you've taken the '@' symbol out? It's referenced elsewhere as '@ip_address' so just wondered if there was a reason for mixing it up